extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveProcessor.java

```diff
@@ -589,6 +589,17 @@ private boolean hasAnnotation(MethodInfo method, short paramPosition, DotName an
                         public Status isJava19OrHigher() {
                             return result;
                         }
+                    })
+                    .setIsDisabledCreator(new Function<>() {
+                        @Override
+                        public Supplier<Boolean> apply(ClassInfo classInfo) {
+                            //TODO: what annotation should be used? @IfBuildProperty, @LookupIfProperty aren't semantically the same...
```

Comment:
what about a property similar for retries:

`io.some.ClassName/methoName/Retry/max-retries=0`

Could something like this work here.. Not sure if this is preferred though..

`io.some.ResourceClass/disabled=true`

Not sure how this works with ENV vars.. Maybe it s a stupid idea..

Reply:
The reason I usually don't like to go down the properties route is that it's hard for users to write them as the IDE will not help you at all.

Reply:
I think that we should introduce a new annotation; e.g. something like `@EndpointDisabled`.